scripts/hypothesistooling.py

```diff
@@ -0,0 +1,79 @@
+import subprocess
+import os
+
+
+def current_branch():
+    return subprocess.check_output([
+        "git", "rev-parse", "--abbrev-ref", "HEAD"
+    ]).decode('ascii').strip()
+
+
+def tags():
+    result = [t.decode('ascii') for t in subprocess.check_output([
+        "git", "tag"
+    ]).split(b"\n")]
+    assert len(set(result)) == len(result)
+    return set(result)
+
+
+ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
I think it's OK, because if we move scripts we'll fail at reading version.py later, but I'm happy to change it.